apps/application-system/api/src/environments/environment.prod.ts

```diff
@@ -25,6 +25,12 @@ export default {
     jwtSecret: process.env.AUTH_JWT_SECRET,
     xRoadBasePathWithEnv: process.env.XROAD_BASE_PATH_WITH_ENV ?? '',
     baseApiUrl,
+    syslumenn: {
+      url: 'https://api.syslumenn.is/api',
+      username: process.env.SYSLUMENN_USERNAME,
+      password: process.env.SYSLUMENN_PASSWORD,
+    },
```

Comment:
This should be used for all api's from syslumenn within the application system. 

This is a valid point since this is pretty specific and currently I don't know how many endpoints we will use from syslumenn.